web/libs/editor/README.md

```diff
@@ -0,0 +1,34 @@
+# Label Studio Frontend
+
+Label Studio Frontend (LSF) is a crucial module of the Label Studio ecosystem, pivotal in driving the entire annotation flow. It's a front-end-only module, combining a user interface for annotation creation with a data layer that standardizes the annotation format. Every manual annotation in Label Studio has been crafted using LSF, making it integral to the system.
+
+### Key Features
+
+Label Studio Frontend is packed with features designed to enhance the annotation process:
+
+- **Customizable Labeling Interfaces:** Create tailored interfaces for different data types like text, images, audio, and video using configurable templates.
+- **Multi-Format Support:** Annotate diverse datasets, including images, text, audio, and video files.
+- **Interactive Annotation Tools:** Utilize tools like bounding boxes, polygons, keypoints, and brush tools for images; text highlight tools for text; and time-series labeling for audio/video.
+- **Real-Time Collaboration:** Enables simultaneous work on projects by multiple annotators.
+- **Data Filtering and Sorting:** Sort and filter data for prioritized tasks or specific data reviews.
+- **Integration with Machine Learning Models:** Connect with ML models for pre-labeling and active learning.
+- **Extensibility and Customization:** Enhance functionality with custom scripts and integrate with external tools.
+
+### Usage Instructions
```

Comment:
Where should we run these instructions from? this dir or the main one I guess?

Reply:
It doesn't matter, you must run it inside web folder or its sub folders, but I can detail it!

Reply:
wow, never thought it's possible! than I like it a lot!